src/setfit/trainer.py

```diff
@@ -7,6 +7,7 @@
 from sentence_transformers.datasets import SentenceLabelDataset
 from sentence_transformers.losses.BatchHardTripletLoss import BatchHardTripletLossDistanceFunction
 from torch.utils.data import DataLoader
+from tqdm import trange
```

Comment:
```suggestion
from tqdm.auto import trange
```
As before. (`tqdm.autonotebook` is also fine)